content/07.pathogenesis.md

```diff
@@ -81,8 +81,9 @@ After interventions, R~t~ decreased by an average of 64% across all countries, w
 The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden (note that this is in part because Sweden did not implement social distancing measures on the same scale as other countries).
 The study concludes that while large changes in R~t~ are observed, it is too early to tell whether the interventions put into place are sufficient to decrease R~t~ below 1.
 
-More generally, population-level epidemic dynamics can be both observed and modelled.
-Data and empirically determined biological mechanisms inform models, while models can be used to try to understand data and systems of interest or to make predictions about possible future dynamics, such as the estimation of capacity needs or the comparison of predicted outcomes among prevention and control strategies.
+More generally, population-level epidemic dynamics can be both observed and modelled [@doi:10.1126/science.abb5659].
+Data and empirically determined biological mechanisms inform models, while models can be used to try to understand data and systems of interest or to make predictions about possible future dynamics, such as the estimation of capacity needs [@doi:10.1073/pnas.2004064117] or the comparison of predicted outcomes among prevention and control strategies[@doi:10.1016/
+S2468-2667(20)30072-4; @doi:10.1073/pnas.2004978117].
```

Comment:
```suggestion
Data and empirically determined biological mechanisms inform models, while models can be used to try to understand data and systems of interest or to make predictions about possible future dynamics, such as the estimation of capacity needs [@doi:10.1073/pnas.2004064117] or the comparison of predicted outcomes among prevention and control strategies[@doi:10/ggp3xq; @doi:10.1073/pnas.2004978117].
```

Switching to the short doi because the full doi has parentheses